### PR TITLE
Feature/expose setting to disable uncaugt exception handler

### PR DESCRIPTION
**Pull request by TroubleMakerBen**
Don't merge. We need to wait for the PLCrashReporter PR to be merged.

### CODE REVIEW

**Comment by TroubleMakerBen** [comment]
@jaelim-ms The PR to PLCrashReporter went through in https://github.com/bitstadium/PLCrashReporter-Private and the version is 1.2.2.
Please build PLCrashReporter from there, drop this in this feature branch and then this is good to go!

File: Vendor/PLCrashReporter/CrashReporter.framework/Versions/A/Headers/PLCrashReporterConfig.h [hunk 1]
```diff
@@ -154,12 +160,19 @@ typedef NS_OPTIONS(NSUInteger, PLCrashReporterSymbolicationStrategy) {
...
+               shouldRegisterUncaughtExceptionHandler: (BOOL) shouldRegisterUncaughtExceptionHandler;
+
+
 /** The configured signal handler type. */
 @property(nonatomic, readonly) PLCrashReporterSignalHandlerType signalHandlerType;
 
 /** The configured symbolication strategy. */
 @property(nonatomic, readonly) PLCrashReporterSymbolicationStrategy symbolicationStrategy;
 
+/** Should PLCrashReporter regiser an uncaught exception handler? This is entended to be used in Xamarin apps */
```
Comment (clpolet) [inline Vendor/PLCrashReporter/CrashReporter.framework/Versions/A/Headers/PLCrashReporterConfig.h]: small typo here 'register'.

Comment (TroubleMakerBen) [inline Vendor/PLCrashReporter/CrashReporter.framework/Versions/A/Headers/PLCrashReporterConfig.h]: and `intended`. Orrrr....that PR was in review dammit.

Comment (jaelim-ms) [inline Vendor/PLCrashReporter/CrashReporter.framework/Versions/A/Headers/PLCrashReporterConfig.h]: This is PLCrashReporter code, not MobileCenter. That will be addressed on their (still our 😄 ) side.

**Pull request merged**